genrl/deep/agents/ddpg/ddpg.py

```diff
@@ -146,6 +155,30 @@ def create_model(self) -> None:
         self.optimizer_policy = opt.Adam(self.ac.actor.parameters(), lr=self.lr_p)
         self.optimizer_q = opt.Adam(self.ac.critic.parameters(), lr=self.lr_q)
 
+    def create_custom_model(self) -> None:
+        """
+        Initialize the custom model
+        Initializes optimizer and replay buffers as well.
+        """
+        if self.noise is not None:
+            self.noise = self.noise(
+                np.zeros_like(action_dim), self.noise_std * np.ones_like(action_dim)
+            )
+        try:
+            self.ac = self.network_type(**kwargs).to(self.device)

```

Comment:
What kwargs are you passing here exactly? Doesn't seem like the function takes any arguments

Reply:
I was thinking that the you can either pass a string as `network_type` or a `class AC(BaseActorCritic)`. The kwargs here then stand for the arguements passed to init `class AC(BaseActorCritic)`